server/rox_options.go

```diff
@@ -54,7 +55,7 @@ func NewRoxOptions(builder RoxOptionsBuilder) model.RoxOptions {
 		logging.SetLogger(NewServerLogger())
 	}
 
-	var dynamicPropertyRuleHandler model.DynamicPropertyRuleHandler
+	var dynamicPropertyRuleHandler model.DynamicPropertyRuleHandler = builder.DynamicPropertyRuleHandler
```

Comment:
you don't need specifying the type here can be `var dynamicPropertyRuleHandler = builder.DynamicPropertyRuleHandler`